### PR TITLE
Update wrap-tool.Rmd

### PR DESCRIPTION
**Pull request by javibio-git**
Added section headers and tentative schedule.

I wonder if something like this could work for the final version of the Cofests! main. Let me know!

### CODE REVIEW

File: wrap-tool.Rmd [hunk 1]
```diff
@@ -4,5 +4,25 @@ Led by: Allie Cliffe, Broad Institute of MIT and Harvard
...
 
+## About
 
-Workflow Description Language (WDL) not in your programming skillset? No problem! Learn to wrap your pipelining tool in WDL and run it in an AnVIL workspace in this CoFest session. We’ll go over WDL basics and how to run pipelines in AnVIL, then walk through how to run your own Unix/Python/R script. Along the way, you’ll write a WDL and build a custom Docker to run it with. Come and wrap a tool of interest! All experience levels are welcome. If time allows, we’ll emerge with some resources for others to run their non-WDL tool in AnVIL. 
+Workflow Description Language (WDL) not in your programming skillset? No problem! Learn to wrap your pipelining tool in WDL and run it in an AnVIL workspace in this CoFest! session. We’ll go over WDL basics and how to run pipelines in AnVIL, then walk through how to run your own Unix/Python/R script. Along the way, you’ll write a WDL and build a custom Docker to run it with. Come and wrap a tool of interest! All experience levels are welcome. We’ll brainstorm and, if time allows, we’ll create resources (tutorials? AnVIL book? Cheat sheet?) for others to use to run their non-WDL tool in AnVIL.
```
Comment (avahoffman) [inline wrap-tool.Rmd:9]: Maybe we could add links? e.g., `[Docker](https://www.docker.com/)`

**Review by avahoffman — APPROVED** [review]

File: wrap-tool.Rmd (hunk 1 above)
Comment (javibio-git) [reply wrap-tool.Rmd:9]: I think that'd be helpful. These are the ones I can think of:

- [WDL](https://openwdl.org/)
- [Docker](https://www.docker.com/)
- [AnVIL](https://anvilproject.org/)

Would that be enough?

Comment (javibio-git) [reply wrap-tool.Rmd:9]: I'll merge this, and I could modify the blurb with the links in a later PR :)

Comment (avahoffman) [reply wrap-tool.Rmd:9]: Sounds good!

**Pull request merged**